exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SchemaUtils.kt

```diff
@@ -102,13 +102,26 @@ object SchemaUtils {
         val toCreate = sortTablesByReferences(tables.toList()).filterNot { it.exists() }
         val alters = arrayListOf<String>()
         return toCreate.flatMap { table ->
-            val (create, alter) = table.ddl.partition { it.startsWith("CREATE ") }
+            val (create, alter) = tableDdlWithoutExistingSequence(table).partition { it.startsWith("CREATE ") }
             val indicesDDL = table.indices.flatMap { createIndex(it) }
             alters += alter
             create + indicesDDL
         } + alters
     }
 
+    private fun tableDdlWithoutExistingSequence(table: Table): List<String> {
+        val autoIncSeq = table.autoIncColumn?.autoIncColumnType?.autoincSeq
+        val sequenceExists = autoIncSeq?.let { currentDialect.sequenceExists(Sequence(it)) } ?: false
+
+        return table.ddl.filter { statement ->
+            if (sequenceExists && autoIncSeq != null) {
+                !statement.lowercase().startsWith("create sequence") || !statement.contains(autoIncSeq)
```

Comment:
If `sequenceExists == true`, `autoIncSeq` could never be null, no? Just to avoid !! in the condition?

Reply:
Yep, you're right. I simplified condition a bit to avoid just-type-checking-condition)